source/adressering-av-undertegner.rst

```diff
@@ -2,76 +2,106 @@
 
-Adressering med fødselsnummer
-===============================
-For å adressere med fødselsnummer må du vite undertegners fødselsnummer, samt e-postadresse og/eller mobilnummer. Denne måten å adressere på gir en sikker identifisering av undertegner, siden kun én bestemt person har mulighet til å åpne og signere dokumentet.
+.. NOTE::
+   Når du som avsender velger adressering med fødselsnummer, kan du velge å inkludere fødselsnummeret i det signerte dokumentet.
 
-Ved :ref:`Signering i portalflyt med fødselsnummer <signering-i-portalflyt-med-fødselsnummer>` må undertegner logge inn i signeringsportalen før han kan se og signere dokumentet. Dette gjør altså at kun riktig person har mulighet til å lese og signere. Se signering i portalflyt, adressering med f.nr her.
+Se :ref:`Signering i portalflyt med fødselsnummer <signering-i-portalflyt-med-fødselsnummer>` for mer informasjon og visuelle eksempler.
 
-Ved :ref:`signering-i-direkteflyt` er undertegner logget inn i avsenders system, og det er da avsender som er ansvarlig for å sende undertegner til riktig lenke i signeringstjenesten.
+Adressering med fødselsnummer og signering i direkteflyt
+--------------------------------------------------------
+Det sendes ikke ut noen automatisk varsling ved signering i direkteflyt. Det er avsenders ansvar å sende undertegner til riktig lenke i signeringstjenesten. 
 
-Når du som avsender velger adressering med fødselsnummer, kan du velge å inkludere fødselsnummeret i det signerte dokumentet.
+Som oftest brukes signering i direkteflyt og adressering med fødselsnummer når undertegner er innlogget i avsenders systemer/nettsider med fødselsnummer.
 
+Se :ref:`signering-i-direkteflyt` for mer informasjon og visuelle eksempler. 
 
-Adressering uten fødselsnummer
+
+2. Adressering på e-post / SMS
 ===============================
 
-Ved adressering uten fødselsnummer blir ikke signeringsoppdraget knyttet til én spesifikk person. Derfor krever signeringstypen ingen innlogging, og du trenger ikke vite undertegners fødselsnummer. I stedet sender du *kun* inn e-postadressen og/eller mobilnummeret som skal motta varsel om signeringen.  Se :ref:`varsler <varslerUtenFødselsnummer>` for å se hvordan varselet ser ut.
+.. NOTE::
+   Dette alternativet er kun tilgjengelig for private virksomheter
 
-Ved :ref:`Signering i portalflyt uten fødselsnummer <signering-i-portalflyt-uten-fødselsnummer>` får undertegner en unik lenke og en kode, og han kan se og signere dokumentet uten innlogging.
+Ved adressering på e-post / SMS blir ikke signeringsoppdraget knyttet til én spesifikk person. Derfor krever signeringstypen ingen innlogging, og du trenger ikke vite undertegners fødselsnummer. I stedet sender du *kun* inn e-postadressen og/eller mobilnummeret som skal motta varsel om signeringen.  Se :ref:`varsler <varslerUtenFødselsnummer>` for å se hvordan varselet ser ut.
```

Comment:
Det er mulig vi potensielt kan skape forvirring her ved å antyde at oppdraget ikke knyttes til person. Det stemmer jo at _oppdraget_ ikke knyttes til spesifikk person (vi ikke har noen mulighet til å enforce dette utover at vedkommende som får tilgang til utsendte varsler vet URL-er for å signere), men dokumenter er jo likefullt sikkert knyttet til personen som faktisk signerer. Det burde kanskje understrekes, da signeringsoppdrag er litt internt domenespråk, og forskjellen på oppdrag og dokument kan kanskje være litt subtil?